wdio/src/configs/wdio.saucelabs.mobile.conf.ts

```diff
@@ -0,0 +1,70 @@
+import type { Options } from '@wdio/types';
+import { config as sauceSharedConfig } from './wdio.saucelabs.shared.conf.ts';
+
+const build = `Sauce Demo Test - ${new Date().getTime()}`;
+
+const iosCapabilitiesShared = {
```

Comment:
Can we simplify this so it's easier for our customers, meaning something like this and only run this with Appium 2

```ts
capabilities: [
  {
    browserName: 'Safari',
    platformName: 'iOS',
    'appium:platformVersion': '16.2',
    'appium:automationName': 'XCUITest',
    'sauce:options': {
      build,
      appiumVersion: '2.0.0'
    },
  },
],
```

Please also do that for Android

Reply:
done, I also changed deviceName to those that are available in us-west and eu